frontend/lib/pages/hp-action-access-for-inspection.tsx

```diff
@@ -5,14 +5,19 @@ import { TextualFormField } from '../form-fields';
 
 const onboardingStepBuilder = new SessionStepBuilder(sess => sess.onboardingInfo);
 
-export const AccessForInspection = onboardingStepBuilder.createStep({
+const AccessForInspectionGenerator = (type?: string) => onboardingStepBuilder.createStep({
```

Comment:
Cool do you mind having the type be `'hp'|'ehp'` and explcitly passing in the argument in the lines below?  that way it'll be super explicit and we'll also be sure that e.g. a typo won't make the logic go awry?  eh it's no biggie.

Reply:
for sure!